src/back/TicketManager/src/Core/TicketManager.Core.Services/Endpoints/Accounts/PasswordValidator.cs

```diff
@@ -8,7 +8,8 @@ public class PasswordValidator : AbstractValidator<string>
     public const int PasswordMinLength = 8;
     public const int PasswordMaxLength = 32;
 
-    public PasswordValidator(int passwordIsTooShortErrorCode,
```

Comment:
Ass you are making changes here please add parameterless constructor as it creates failes

![image](https://user-images.githubusercontent.com/43820607/227800274-6498ff96-10f0-4ba4-b188-18f2933438ad.png)